src/region_cone.cpp

```diff
@@ -187,6 +311,14 @@ RegCone::~RegCone()
   delete[] contact;
 }
```

Comment:
This seems to be missing `delete[]` statements for duplicated variable names and thus leaking memory.

Reply:
you are right. I added the missing statements.